components/scream/p3/p3.hpp

```diff
@@ -1,12 +1,42 @@
 #ifndef SCREAM_P3_HPP
 #define SCREAM_P3_HPP
 
-namespace Scream {
-namespace P3 {
+#include "scream_kokkos.hpp"
+#include "scream_types.hpp"
 
-int p3_stub();
+namespace scream {
+namespace p3 {
 
-}  // namespace P3
-}  // namespace Scream
+template <typename exespace = Kokkos::DefaultExecutionSpace,
```

Comment:
Remove this generality and rename to FortranData. The C++ data should group all arrays in a column together so the data are close; hence, multi-D arrays over (col,lev) won't be used. Moreover, the lev dim should be compile time. In summary, this current Data should be for Fortran calling only.